pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -537,6 +530,8 @@ func getConsoleTemplate(cr *api.KieApp) api.ConsoleTemplate {
 		if cr.Status.Applied.Environment == api.RhpamAuthoringHA || cr.Status.Applied.Environment == api.RhdmAuthoringHA {
 			if cr.Status.Applied.Objects.Console.DataGridAuth != nil {
 				template.DataGridAuth = *cr.Status.Applied.Objects.Console.DataGridAuth
+			} else {
+				template.DataGridAuth = api.DataGridAuth{}
```

Comment:
This change is not included in PR for changes on main branch - see #779

Reply:
Fixed